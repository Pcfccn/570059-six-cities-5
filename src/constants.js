const Path = {
  MAIN: `/`,
  SIGN_IN: `/login`,
  FAVORITES: `/favorites`,
  OFFER: `/offer/:id`,
  getUrlById: (id) => (`/offer/${id}`)
};

const ratingStars = [5, 4, 3, 2, 1];
const ratingInputTitle = [`perfect`, `good`, `not bad`, `badly`, `terribly`];

const widthStartsCoefficient = 20;

const CityCoordinates = {
  Amsterdam: [52.38333, 4.9],
  Paris: [48.864716, 2.349014],
};

const OfferCardClassName = {
  nearPlace: `near-places__card`,
  citiesPlace: `cities__place-card`,
};

const MapClassName = {
  property: `property__map`,
  cities: `cities__map`,
};

const cities = [`Paris`, `Cologne`, `Brussels`, `Amsterdam`, `Hamburg`, `Dusseldorf`];

export {Path, widthStartsCoefficient, ratingStars, ratingInputTitle, CityCoordinates, OfferCardClassName, MapClassName, cities};
