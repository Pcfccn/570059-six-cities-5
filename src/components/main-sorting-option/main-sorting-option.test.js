import React from "react";
import renderer from "react-test-renderer";
import {SortType} from "../../constants";
import MainSortingOption from "./main-sorting-option";

const noop = () => {};

describe(`MainSortingOption`, () => {
  it(`Should render correctly`, () => {
    const tree = renderer
    .create(<MainSortingOption
      activeSortType={SortType.POPULAR_DESC}
      changeType={noop}
      type={SortType.POPULAR_DESC}
    />)
    .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
