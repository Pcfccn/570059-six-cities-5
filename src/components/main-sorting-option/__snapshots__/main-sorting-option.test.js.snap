// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainSortingOption Should render correctly 1`] = `
<li
  className="places__option places__option--active"
  onClick={[Function]}
  tabIndex="0"
>
  Popular
</li>
`;
